RELEASES.md

```diff
@@ -1,3 +1,9 @@
+# Version 0.9.0
+
+- Support Proxies through environment variables (#809, #851)
+- Avoid unnecessary `exists` calls for files (#834)
+- Rework package installs to allow for directly calling package manager (#848, #849)
+
```

Comment:
If I understood the changes and read [the updated docs](https://github.com/volta-cli/website/pull/81/files#diff-c1a34c7015d9c7909962653b885f64b8458a3c9dcae05df63d7aa4ef3e2403f3R43) correctly, the changes to hooks *are* breaking changes and we should make a point to call them out here!

Reply:
Good call! I included it in the blog post, but not here. I'll add that!